tests/acceptance/features/apiAccountsHashDifficulty/assignRole.feature

```diff
@@ -62,3 +62,29 @@ Feature: assign role
       | Space Admin | Space Admin | 400        | User         |
       | User        | Admin       | 400        | User         |
       | User        | Space Admin | 400        | User         |
+
+  Scenario Outline: assign role to the user using graph api
+    Given user "Alice" has been created with default attributes and without skeleton files
+    And the administrator has given "Alice" the role "<userRole>" using the Graph API
+    When user "Alice" retrieves assigned role using the Graph API
+    Then the HTTP status code should be "200"
+    And the Graph API response should have the role "<userRole>"
+    Examples:
+      | userRole    |
+      | Admin       |
+      | Space Admin |
+      | User        |
+      | Guest       |
+
+  Scenario Outline: assign role to the user with setting api and list role with graph api
+    Given user "Alice" has been created with default attributes and without skeleton files
+    And the administrator has given "Alice" the role "<userRole>" using the Graph API
+    When user "Alice" tries to get list of assignment
+    Then the HTTP status code should be "201"
+    And the setting API response should have the role "<userRole>"
+    Examples:
+      | userRole    |
+      | Admin       |
+      | Space Admin |
+      | User        |
+      | Guest       |
```

Comment:
how about moving this to grapAPI suite ? this feature seems like of `setting api`